.eslint-bin/pre-commit-hook.js

```diff
@@ -3,6 +3,12 @@ const simpleGit = require('simple-git')(`${__dirname}/..`);
 
 let listFilesModified = [];
 
+function excludeNonCommitedFiles(file) {
```

Comment:
Why ? 🤔

Reply:
The user working copy belongs to the userland.
`pre-commit-hook` job is to control files that the user is going to send into the commit, not everything present in the working copy.